src/Organic/PageInjection.php

```diff
@@ -193,6 +194,12 @@ public function injectBrowserSDK() {
         }
     }
 
+    public function injectOrganicCustomCss() {
+        $siteId = $this->organic->getSiteId();
+        $cssUrl = $this->organic->getRestAPIUrl() . '/sdk/customcss/' . $siteId;
+        wp_enqueue_style( $siteId, $cssUrl, [], '1.0' );
```

Comment:
Not sure what to do with this version number, open to suggestions.

Reply:
I guess the question is how important is cache invalidation for this CSS? If, as I suspect, it is important, then we probably need to do something else. Maybe @sergunich has better ideas, but here are a couple options:
- Using a `custom_css_version` field (an int, or a short timestamp) that updates whenever the `custom_css` field is updated.
- Having an `sdk_version` field that updates whenever the sdk config is changed. (More general than the above, but also necessarily less granular. I don't know how often sdk config changes are made; if frequently, then the above is probably better.)

Reply:
And the first discussion post [here](https://developer.wordpress.org/reference/functions/wp_enqueue_style/) lists a solution as well.

Reply:
@Mwindo I'm not sure the solution in the discussion post will work for us because we'd need last modified time for a remote file (technically just a text response). There are some implementations of this, but they seem overly complicated compared to your other proposals.

Reply:
I just realized I didn't see this other PR. It looks like maybe caching was already taken care of (similar to my suggestion in part 2). https://github.com/orgnc/platform/pull/3240/files#r1161824120. If so, it doesn't really matter what version we pass in I don't think.

Reply:
Yes, we don't need any special cache handling here because it will work similar to SDK serving (cache and invalidation handled by Fastly)